Packs/Workday/Integrations/WorkdaySignOnEventCollector/WorkdaySignOnEventCollector.py

```diff
@@ -0,0 +1,565 @@
+
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+VENDOR = "workday"
+PRODUCT = "sign_on"
+API_VERSION = "v40.0"
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+def get_from_time(seconds_ago: int) -> str:
+    current_time = datetime.now(tz=timezone.utc)
+    from_time = current_time - timedelta(seconds=seconds_ago)
+    return from_time.strftime(DATE_FORMAT)
+
+
+def fletcher16(data: bytes) -> int:
+    """
+    Compute the Fletcher-16 checksum for the given data.
+
+    The Fletcher-16 checksum is a simple and fast checksum algorithm that provides
+    a checksum value based on the input data. It's not as collision-resistant as
+    cryptographic hashes but is faster and can be suitable for non-security-critical
+    applications.
+
+    Parameters:
+    - data (bytes): The input data for which the checksum is to be computed.
+
+    Returns:
+    - int: The computed Fletcher-16 checksum value.
+    """
+    sum1, sum2 = 0, 0
+    for byte in data:
+        sum1 = (sum1 + byte) % 256
+        sum2 = (sum2 + sum1) % 256
+    return (sum2 << 8) | sum1
+
+
+def generate_checksum(event: dict) -> str:
```

Comment:
Please change to generate_event_pseudo_id() because I think it better explains what it is ment to do, and it is not just checksum.